webui/react/src/pages/Settings/UserManagement.tsx

```diff
@@ -48,21 +51,24 @@ const UserActionDropdown = ({ fetchUsers, user, groups }: DropdownProps) => {
 
   enum MenuKey {
     EDIT = 'edit',
-    STATE = 'state'
+    STATE = 'state',
+    VIEW = 'view'
   }
 
   const funcs = {
     [MenuKey.EDIT]: () => { onClickEditUser(); },
     [MenuKey.STATE]: () => { onToggleActive(); },
+    [MenuKey.VIEW]: () => { onClickViewUser(); },
```

Comment:
can we call `openEditUserModal(true)` in here to avoid creating `onClickViewUser` for one use? Same for `onClickEditUser`.
I think `onToggleActive` is OK / can stay because it's more complex

Reply:
Sure thank you for pointing that out